compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.wso2.ballerinalang.compiler.semantics.analyzer;
 
+import io.ballerina.runtime.api.flags.SymbolFlags;
```

Comment:
We shouldn't use runtime classes here. 

You can use org.wso2.ballerinalang.compiler.semantics.model.symbols.Symbols#isFlagOn and org.wso2.ballerinalang.util.Flags#CONFIGURABLE instead.